.editorconfig

```diff
@@ -0,0 +1,17 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# editorconfig.org
+
+root = true
```

Comment:
Argh -- I just realized that (1) root=true means that editorconfig will stop looking for higher directory editorconfig settings.

Also (2) it seems that local settings override higher level settings!  This is going to be a bit or a problem for the accessibility use case.  I'm going to spend a few cycles trying to find a solution.

Reply:
All right -- it appears that indeed project settings override global settings and I don't think there is going to be a solution here any time soon.  We can talk about this later as to whether it means we want to avoid use of editorconfig overall; in the mean time I do think that use of tabs and editorconfig is a reasonable compromise since folks who want to ignore editorconfig entirely have the means to do so.